src/config/Option.test.ts

```diff
@@ -0,0 +1,181 @@
+import { CodePointBuffer, Parameter } from "../token/Tokens";
+import { BooleanOption, EnumOption, RangeOption } from "./Option";
+
+test("BooleanOption Get", () => {
```

Comment:
change to `BooleanOption get & getter function`

Reply:
Updated, please refer to 
[c14362b](https://github.com/Jerrylum/protocol-diagram/pull/8/commits/c14362b46accd850aaa7de3765b4c67e42df8132)